rjs/constants.go

```diff
@@ -50,6 +50,9 @@ const (
 	ReJSONCommandDEBUG     ReJSONCommandID = 17
 	ReJSONCommandFORGET    ReJSONCommandID = 18
 	ReJSONCommandRESP      ReJSONCommandID = 19
+	ReJSONCommandQGET      ReJSONCommandID = 20
+	ReJSONCommandINDEXADD  ReJSONCommandID = 21
+	ReJSONCommandSETINDEX  ReJSONCommandID = 22
```

Comment:
Just a suggestion to differentiate from others
```suggestion
	ReJSON2CommandQGET      ReJSONCommandID = 20
	ReJSON2CommandINDEXADD  ReJSONCommandID = 21
	ReJSON2CommandSETINDEX  ReJSONCommandID = 22
```